assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -218,13 +226,26 @@ function SingleSelectionMovable( {
 			pinchable={ true }
 			keepRatio={ 'image' === selectedElement.type && isResizingFromCorner }
 			renderDirections={ ALL_HANDLES }
+			snappable={ true }
```

Comment:
Are you planning to do the same with `MultiSelectionMovable` later? If so, please rename the PR to reflect this.

Reply:
Done